### DISCUSSION TITLE
热力图问题

### DISCUSSION THREAD

**Discussion by sassywp** [zouyaoji/vue-cesium#132] [General]
热力图数据我照着vue3版本的的成都的数据改成了其他城市的坐标，然后没有加载上，请问标准数据是怎么得来的，坐标格式是什么的格式呢？

**Comment by zouyaoji** [comment]
@sassywp  还有个范围要写，rectangle参数。

**Reply by sassywp** [reply]
好的 非常感谢 大佬流批orz

**Reply by deronendless** [reply]
@zouyaoji rectangle参数如何获取呢，是热力图数据的上下左右吗？

**Comment by xkk1011** [comment]
你好，请问一下为什么我的海域数据是{
      "x": 129.2291717529297,
      "y": 21.645832061767578,
      "value": 0.15394999086856842
    },
    {
      "x": 129.2708282470703,
      "y": 21.645832061767578,
      "value": 0.1545500010251999
    },
    {
      "x": 129.3125,
      "y": 21.645832061767578,
      "value": 0.15497499704360962
    },
    {
      "x": 129.3541717529297,
      "y": 21.645832061767578,
      "value": 0.15519998967647552
    },
    {
      "x": 129.3958282470703,
      "y": 21.645832061767578,
      "value": 0.155349999666214
    },
    然而实际加载的时候，控制台会提示有
    vue-cesium.js?v=df17f227:5911 Uncaught (in promise) IndexSizeError: Failed to execute 'addColorStop' on 'CanvasGradient': The provided value (-0.702593) is outside the range (0.0, 1.0).
